types.d.ts

```diff
@@ -0,0 +1,366 @@
+import {EventEmitter} from 'events';
+import * as SocketIO from 'socket.io';
+import {IRouter, RequestHandler} from 'express-serve-static-core';
+
+/**
+ * NodeCG instance
+ */
+interface NodeCGCommon<P extends Platform, M = SendMessageReturnType<P>> {
+	bundleName: string;
+	bundleConfig: any;
+	bundleVersion: string;
+	readonly bundleGit: {
+		branch: string;
+		hash: string;
+		shortHash: string;
+		date?: Date;
+		message?: string;
+	};
+	Logger(name: string): typeof Logger;
```

Comment:
It was because the previous typedef made the `Logger` property itself a method that returns `Logger` constructor, while it was supposed to be the constructor itself.

Should be fixed now. Also added test corresponding to it.